lang/src/test/java/org/kaazing/k3po/lang/internal/parser/ScriptParserImplTest.java

```diff
@@ -1259,6 +1282,28 @@ public void shouldParseAcceptScript() throws Exception {
     }
 
     @Test
+    public void shouldParseAcceptScriptWithOptions() throws Exception {
+
+        String script =
+                "# tcp.client.accept-then-close\n" +
+                "accept option transport \"localhost:8000\"\n" +
+                "accept tcp://localhost:7788\n" +
```

Comment:
Let's discuss the syntax a bit more.

It's starting to feel like an imperative programming language when we setup the `option` state and then issue the `accept` or `connect`.  We have been deliberately trying to avoid that, for example no looping in K3PO script language.

Would have anticipated the options to be either part of the accept itself or after the accept (but before accepted). What are your thoughts on that?